components/BlogCard/blog.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import BlogCard from "./BlogCard";
+
+
+const blogs = [
+  {
+    title: "Зошто моето куче јаде трева?",
+    text: "Lorem ipsum dolor sit amet, consectetur adipisicing elit. Earum error illum temporibus quis ducimus.",
+    imageUrl: "./BlogImg/blogImg3.svg",
+    time: "26.6.2024",
```

Comment:
variable name should be `date`

Reply:
While you changed the BlogCardProps, you forgot to change the variables in the data.